app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def bootstrap_class_for(type)
+    { success: 'alert-success',
+      error:   'alert-danger',
+      alert:   'alert-warning',
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.